helm/drone-fork-approval-extension/values.yaml

```diff
@@ -0,0 +1,43 @@
+secret: ""
+
+replicaCount: 1
+
+image:
+  repository: quay.io/gravitational/drone-fork-approval-extension
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+  hosts:
+    - host: drone-fork-approval-extension.local
+      paths:
+      - path: /
+        backend:
+          serviceName: drone-fork-approval-extension.local
+          servicePort: 80
+  tls: []
```

Comment:
Remove this part if removing `Ingress` too.

Reply:
Removed in 636981a.